docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/user-and-session-identification/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "User and session identification and stitching"
```

Comment:
if it is called stitching we should all out that they should track user_ids because that will guarantee out of the box user stitching for modelling. otherwise i would remove it from the name